patches/03.disk-widget.patch

```diff
@@ -10,7 +10,7 @@
 +    } else {
```

Comment:
Why are you modifying disk widget in a temperature widget creation pull request?